numpy/lib/function_base.py

```diff
@@ -2921,10 +2921,13 @@ def i0(x):
 
     See Also
     --------
-    scipy.special.iv, scipy.special.ive
+    scipy.special.i0, scipy.special.iv, scipy.special.ive
 
     Notes
     -----
+    The scipy implementation is recommended over this function:
+    It a proper ufunc, and more than an order of faster.
```

Comment:
Yes, I'd like to order three more fasters for ufuncs on scalars please.

I think this needs a few more words.